mailbox/spring/src/main/resources/META-INF/spring/spring-mailbox.xml

```diff
@@ -60,11 +60,4 @@
      -->
     <bean id="messageParser" class="org.apache.james.mailbox.store.mail.model.impl.MessageParser"/>
 
-
-
-    <beans profile="cassandra">
-        <import resource="classpath:META-INF/spring/mailbox-cassandra.xml"/>
-        <import resource="classpath:META-INF/spring/mailbox-elasticsearch.xml"/>
```

Comment:
You don't remove the no more references files: mailbox-cassandra.xm, mailbox-elasticsearch.xml, kafka.xml and cassandra-session.xml?